apps/andi/lib/andi_web/live/dataset_live_view.ex

```diff
@@ -0,0 +1,46 @@
+defmodule AndiWeb.DatasetLiveView do
+  use Phoenix.LiveView
+
+  def render(assigns) do
```

Comment:
This doesn't have to go here. 

```elixir
def render(assigns) do
  AndiWeb.DatasetView.render("index.html")
end
```
That code should render it from a separate template file.

Reply:
You should be able to just put it in a .leex file and have the render function be generated, right?

Reply:
We're going to use LiveComponents to break this up, but still keep the template in the files.